packages/contracts-bedrock/test/Authorization.t.sol

```diff
@@ -0,0 +1,266 @@
+        _addSpec("OptimismPortal", _getSel("SYSTEM_CONFIG()"), false);
+        _addSpec("OptimismPortal", _getSel("depositTransaction(address,uint256,uint64,bool,bytes)"), false);
+        _addSpec("OptimismPortal", _getSel("donateETH()"), false);
+        _addSpec("OptimismPortal", OptimismPortal.finalizeWithdrawalTransaction.selector, false);
+        _addSpec("OptimismPortal", _getSel("finalizedWithdrawals(bytes32)"), false);
+        _addSpec("OptimismPortal", _getSel("guardian()"), false);
+        _addSpec("OptimismPortal", _getSel("initialize(address)"), false);
+        _addSpec("OptimismPortal", _getSel("isOutputFinalized(uint256)"), false);
+        _addSpec("OptimismPortal", _getSel("l2Oracle()"), false);
+        _addSpec("OptimismPortal", _getSel("l2Sender()"), false);
+        _addSpec("OptimismPortal", _getSel("minimumGasLimit(uint64)"), false);
+        _addSpec("OptimismPortal", _getSel("params()"), false);
+        _addSpec("OptimismPortal", _getSel("paused()"), false);
+        _addSpec("OptimismPortal", OptimismPortal.proveWithdrawalTransaction.selector, false);
+        _addSpec("OptimismPortal", _getSel("provenWithdrawals(bytes32)"), false);
+        _addSpec("OptimismPortal", _getSel("spacer_53_0_1()"), false);
+        _addSpec("OptimismPortal", _getSel("superchainConfig()"), false);
+        _addSpec("OptimismPortal", _getSel("systemConfig()"), false);
+        _addSpec("OptimismPortal", _getSel("version()"), false);
+
+        // ProtocolVersions
+        _addSpec("ProtocolVersions", _getSel("RECOMMENDED_SLOT()"), false);
+        _addSpec("ProtocolVersions", _getSel("REQUIRED_SLOT()"), false);
+        _addSpec("ProtocolVersions", _getSel("VERSION()"), false);
+        _addSpec("ProtocolVersions", ProtocolVersions.initialize.selector, false);
+        _addSpec("ProtocolVersions", _getSel("owner()"), false);
+        _addSpec("ProtocolVersions", ProtocolVersions.recommended.selector, false);
+        _addSpec("ProtocolVersions", _getSel("renounceOwnership()"), true);
+        _addSpec("ProtocolVersions", ProtocolVersions.required.selector, false);
+        _addSpec("ProtocolVersions", ProtocolVersions.setRequired.selector, true);
+        _addSpec("ProtocolVersions", ProtocolVersions.setRecommended.selector, true);
+        _addSpec("ProtocolVersions", _getSel("transferOwnership(address)"), false);
+        _addSpec("ProtocolVersions", _getSel("version()"), false);
+
+        // ResourceMetering
+        _addSpec("ResourceMetering", _getSel("params()"), false);
+
+        // SuperchainConfig
+        _addSpec("SuperchainConfig", _getSel("GUARDIAN_SLOT()"), false);
+        _addSpec("SuperchainConfig", _getSel("PAUSED_SLOT()"), false);
+        _addSpec("SuperchainConfig", _getSel("guardian()"), false);
+        _addSpec("SuperchainConfig", _getSel("initialize(address)"), false);
+        _addSpec("SuperchainConfig", _getSel("pause(string)"), false);
+        _addSpec("SuperchainConfig", _getSel("paused()"), false);
```

Comment:
These should have auth of `true`, as should `deleteL2Outputs`, `proposeL2Output`, functions with the `onlyOtherBridge` modifier, and likely some others. Searching for `msg.sender ==` or `== msg.sender` is probably a good way to find most of these. (Totally understand this is a draft so maybe you didn't go through them all yet).

For the ones that are `true` we should also manually a corresponding auth test is present (so far I've confirmed this for `pause` and `unpause` only)